starter-code/routes/index.js

```diff
@@ -0,0 +1,40 @@
+var express = require('express');
+var router = express.Router();
+const mongoose = require('mongoose');
+const Placemodel = require('../models/place')
+// view engine setup
+/* GET home page. */
+
+router.get('/', (req, res, next) => {
+  Placemodel.find({}, (err, Place) => {
```

Comment:
With find you are retrieving an array of places. Therefore, naming the argument `Place` is probably not a good idea. It start with an uppercase letter which suggest a class (although it isn't) and it is singular which suggest a single object (although it is an array of objects).